src/commands.rs

```diff
@@ -17,3 +22,31 @@ pub(crate) mod upgrade;
 
 #[cfg(feature = "pczt-qr")]
 pub(crate) mod keystone;
+
+pub(crate) fn select_account<DbT: WalletRead<AccountId = AccountUuid>>(
+    db_data: &DbT,
+    account_uuid: Option<Uuid>,
+) -> Result<DbT::Account, anyhow::Error>
+where
+    DbT::Error: std::error::Error + Sync + Send + 'static,
+{
+    let account_id = match account_uuid {
+        Some(uuid) => Ok(AccountUuid::from_uuid(uuid)),
+        None => {
+            let account_ids = db_data.get_account_ids()?;
+            if account_ids.len() > 1 {
+                Err(anyhow!(
+                    "More than one account is available; please specify the account UUID."
+                ))
+            } else {
+                account_ids
+                    .first()
+                    .map_or(Err(anyhow!("Wallet contains no accounts.")), |v| Ok(*v))
+            }
```

Comment:
I think slice matching would be cleaner:
```suggestion
            match &account_ids[..] {
                [] => Err(anyhow!("Wallet contains no accounts.")),
                [account_id] => Ok(*account_id),
                _ => Err(anyhow!(
                    "More than one account is available; please specify the account UUID."
                )),
            }
```